### ISSUE TITLE
Running Marathon on macOS Catalina does not work

### ISSUE THREAD

**Issue by LambdaDigamma** [JohnSundell/Marathon#206]
**Platform**
macOS Catalina Version 10.15 Beta (19A512f)

**Issue**
I freshly installed Marathon the recommended way on macOS Catalina Beta Version. 
It seems to install without any error, but when trying to run Marathon via `marathon` I get the following error:
`This copy of libswiftCore.dylib requires an OS version prior to 10.14.4.`
`[1]    13259 abort      marathon run`

I didn't dig into source code yet, but does anybody know a fast way to solve this error?
Thanks in advance!

**Comment by LambdaDigamma** [comment]
I resolved this error by switching Command Line Tools to use Xcode 11 Beta: 
`sudo xcode-select --switch /Applications/Xcode-beta.app/Contents/Developer` 

Please excuse the inconvenience!

**Comment by johndpope** [comment]
I'm seeing  this error - although at some stage it was working fine. 

cd "/Users/johndpope/.marathon/Scripts/Cache/-Users-johndpope-Downloads-frontpage-ios-app-FrontPage-src-Sync/" && readlink OriginalFile
   /Users/johndpope/Downloads/frontpage-ios-app/FrontPage/src/Sync.swift
   $ cd "/Users/johndpope/Downloads/frontpage-ios-app/FrontPage/src/" && /usr/bin/env xcrun --sdk macosx swift package --version
   Apple Swift Package Manager - Swift 5.1.0 (swiftpm-14672.3)
   $ cd "/Users/johndpope/.marathon/Scripts/Cache/-Users-johndpope-Downloads-frontpage-ios-app-FrontPage-src-Sync/" && /usr/bin/env xcrun --sdk macosx swift build -C /Users/johndpope/.marathon/Scripts/Cache/-Users-johndpope-Downloads-frontpage-ios-app-FrontPage-src-Sync/
   [1/3] Compiling Sync main.swift
   <unknown>:0: error: invalid value '5.1' in '-swift-version 5.1'
   <unknown>:0: note: valid arguments to '-swift-version' are '4', '4.2', '5'
   warning: '--chdir/-C' option is deprecated; use '--package-path' instead
💥  Failed to compile script

**Comment by LambdaDigamma** [comment]
You are right. After successful installation I wasn't successful in building and compiling a Marathon Script on Catalina either.

**Comment by jstewuk** [comment]
Installed without an issue but unable to complete helloWorld example.  Tests also failing.
XCode11b6, Catalina beta "10.15 Beta (19A536g)". Haven't had time to chase.

**Comment by JohnSundell** [comment]
Hi everyone, please note that Marathon [is being deprecated](https://github.com/JohnSundell/Marathon/issues/208) and will not be updated for macOS Catalina (at least not by me). Feel free to either debug this and submit a PR, or migrate your scripts to use SwiftPM directly (which is my personal recommendation).
